R/methods-pSet.R

```diff
@@ -546,10 +546,29 @@ setReplaceMethod("$", "pSet", function(x, name, value) {
 })
 
 setReplaceMethod("pData", "pSet", function(object, value) {
+    if (!is.data.frame(value))
```

Comment:
Is this necessary? Trying `pData(obj) <- x` with anything else than a `data.frame` leads to an error
```
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘pData<-’ for signature ‘"NAnnotatedDataFrame", "not.a.data.frame"’
```

Reply:
I know - I just wanted to throw a more user-friendly error. There might be users not understanding what the *default* error message means.

Reply:
Ok then.